CMakeLists.txt

```diff
@@ -24,6 +24,11 @@ install(FILES "inc/azure_macro_utils/macro_utils.h" DESTINATION "${CMAKE_INSTALL
 
 if("${CMAKE_VERSION}" VERSION_GREATER 3.0.0)
     add_library(azure_macro_utils_c INTERFACE)
+    target_include_directories(azure_macro_utils_c
+        INTERFACE
+            $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/inc>
+            $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/azure_macro_utils>
+    )
     install(TARGETS azure_macro_utils_c EXPORT azure_macro_utils_cTargets
```

Comment:
Can we drop the install thing for the love of everything? It was only needed for Edge v1. Who else needs it?
And Vcpkg should have it's own CMakeLists (think CMakeLists.vcpkg, which can include CMakeLists.Txt if it wants but let's please separate this out as it's becoming a nightmare.

Reply:
that would have to be a different PR. not having this generator expression breaks the cmake generation/build. but i mostly agree.